templates/managed/kafka.html

```diff
@@ -140,6 +140,45 @@ <h3>
   </div>
 </section>
 
+<section class="p-strip--suru-accent is-dark" id="webinar">
+  <div class="row">
+    <div class="col-4 u-hide--small">
+      <div class="p-card">
+        <a href="https://www.brighttalk.com/webcast/6793/439718" aria-hidden="true" tabindex="-1">
```

Comment:
This links to the Cassandra webinar not the Kafka one

Reply:
Done, thanks